toolchain/sem_ir/formatter.cpp

```diff
@@ -296,14 +297,14 @@ class NodeNamer {
                                   *this, parse_node, std::move(name))};
   }
 
-  // Finds and adds a suitable block label for the given semantics node that
+  // Finds and adds a suitable block label for the given sem_ir instruction that
```

Comment:
Slight preference for:
```suggestion
  // Finds and adds a suitable block label for the given SemIR instruction that
```

Or:
```suggestion
  // Finds and adds a suitable block label for the given semantics IR instruction that
```